README.md

```diff
@@ -22,12 +22,13 @@ You can see the version history [here](RELEASE_NOTES.md).
 - FSharp.Core 4.3.1.0 (F# 3.1)
 - FSharp.Core 4.4.0.0 (F# 4.0)
 - FSharp.Core 4.4.1.0 (F# 4.1)
+- FSharp.Core 4.4.3.0 (F# 4.1 update)
 
 ## Supported platforms
 
-- .NET Framework 4.5
-- PCL profile 7 (.NET Framework 4.5, Windows 8)
-- PCL profile 259 (.NET Framework 4.5, Windows 8, Windows Phone 8.1, Windows Phone Silverlight 8)
+- .NET Framework 4.x
```

Comment:
4.0 is not supported, only 4.5 and 4.6, right? So can you change this to 4.5+

Reply:
Done thanks